src/articles/articles.controller.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Controller,
+  Get,
+  Put,
+  Delete,
+  Param,
+  ParseIntPipe,
+  Post,
+  UseGuards,
+  Body,
+} from '@nestjs/common';
+import { ArticlesService } from './articles.service';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { CreateArticleDto } from './dto/create-article.dto';
+import { GetUser } from 'src/auth/decorator/get-user.decorator';
+import { AuthGuard } from '@nestjs/passport';
+import { User } from 'generated/prisma';
+
+@Controller('api')
```

Comment:
```suggestion
@Controller('api/articles')
```

có nên move path `articles` lên đây,
và xóa ở từng controller riêng lẻ đi ko em nhỉ?